pkg/datarepair/checker/config.go

```diff
@@ -20,13 +21,18 @@ import (
 
 // Config contains configurable values for checker
 type Config struct {
-	QueueAddress string        `help:"data checker queue address" default:"redis://127.0.0.1:6378?db=1&password=abc123"`
-	Interval     time.Duration `help:"how frequently checker should audit segments" default:"30s"`
+	QueueAddress     string        `help:"data checker queue address" default:"redis://127.0.0.1:6378?db=1&password=abc123"`
+	Interval         time.Duration `help:"how frequently checker should audit segments" default:"30s"`
+	IrreparabledbURL string        `help:"the database connection string to use" default:"$CONFDIR/irreparable.db"`
```

Comment:
See how other databases are handling this. It should be specified as `postgres://xyz:13431/XXX` or `sqlite://file:memory` etc.

Reply:
done.